src/lizaalert/courses/admin.py

```diff
@@ -54,12 +54,20 @@ class ChapterInline(admin.TabularInline):
         "order_number",
         "title",
         "deleted_at",
+        "user_created",
+        "user_modifier",
     )
 
     # Метод для отображения ссылки на главу курса
     def get_chapter_link(self, obj):
-        url = reverse("admin:courses_chapter_change", args=(obj.id,))
-        return format_html('<a href="{}">{}</a>', url, obj.title)
+        id = obj.id
+        title = obj.title
+        url = reverse("admin:courses_chapter_change", args=(id,))
+        if id is None:
+            return "Новая глава"
+        if title is None:
+            title = "Нет названия"
```

Comment:
если пустая строка, то тоже лучше выполнить называние

```suggestion
        title = title or "Нет названия"
```